### DISCUSSION TITLE
> character does not get escaped?

### DISCUSSION THREAD

**Discussion by mehulkar** [prysk/prysk#201] [Q&A] [answered]
I have a test:

```
  $ runcommand
  > some output
  > another output
```

Prysk fails to validate because it's trying to execute `some output`

```
+  /bin/sh: line 9: some: command not found
+  /bin/sh: line 10: another: command not found
+  [127]
```

The docs note that `>` is for multiline commands. Is there anyway to escape this character? My output starts with this unfortunately. I also tried adding ` (esc)` to the end of the line

For a more concrete use case, my `runcommand` is invoking `npm run build`, and this is how `npm` prints the script it's about to run.

**Comment by nigels-com** [comment, marked as answer]
Perhaps as a workaround:
`\> some output (re)`

**Comment by mehulkar** [comment]
That worked, thank you! Feel free to close this
